content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
-Customers --> Batch Payments` or :menuselection:`Accounting --> Vendors --> Batch Payments`. In the
-list view of payments, you can select and group several payments in a batch by clicking
-:menuselection:`Action --> Create Batch Payment`.
+Batch payments allow to group payments from multiple partners to ease
+:doc:`reconciliation <bank/reconciliation>`. They are also useful when depositing :doc:`checks
+<payments/checks>` to the bank or for :doc:`SEPA payments <payments/pay_sepa>`. To do so, go to
+:menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
+Vendors --> Batch Payments`. In the list view of payments, to select and group several payments in a
+batch, click :menuselection:`Action --> Create Batch Payment`.
 
 .. seealso::
   - :doc:`payments/batch`
   - :doc:`payments/batch_sdd`
 
-.. _payments/matching:
+.. _accounting/payments/matching:
 
 Payments matching
 -----------------
 
 The :guilabel:`Payments matching` tool opens all unreconciled customer invoices or vendor bills and
-allows you to process them individually, matching all payments and invoices in one place. You can
-reach this tool from the :menuselection:`Accounting Dashboard --> Customer Invoices / Vendor Bills`,
-click the drop-down menu button (:guilabel:`⋮`), and select :guilabel:`Payments Matching`, or go to
-:menuselection:`Accounting --> Accounting --> Reconciliation`.
+allows to process them individually, matching all payments and invoices in one place. Go to the
+:guilabel:`Accounting Dashboard`, and click the :icon:`fa-ellipsis-v` drop-down menu button from the
+:guilabel:`Customer Invoices` or :guilabel:`Vendor Bills` journals, and select
+:guilabel:`Payments Matching`, or go to :menuselection:`Accounting --> Accounting -->
+Reconciliation`.
 
 .. image:: payments/payments-journal.png
    :alt: Payments matching menu in the drop-down menu.
 
 .. note::
    During the :doc:`reconciliation <bank/reconciliation>`, if the sum of the debits and credits does
    not match, there is a remaining balance. This either needs to be reconciled at a later date or
-   needs to be written off directly.
+   written off directly.
+
+.. _accounting/payments/batch-payments-matching:
 
 Batch payments matching
 -----------------------
 
-You can use the batch reconciliation feature to reconcile several outstanding payments or invoices
+Use the batch reconciliation feature to reconcile several outstanding payments or invoices
 simultaneously for a specific customer or vendor. Go to :menuselection:`Accounting --> Reporting -->
-Aged Receivable / Aged Payable`. You can see all transactions that have not been reconciled yet for
-that partner, and when you select a customer or vendor, the :guilabel:`Reconcile` option is
+Aged Receivable / Aged Payable`. All transactions that have not yet been reconciled for that
+partner are visible, and when a customer or vendor is selected, the :guilabel:`Reconcile` option is
 displayed.
```

Comment:
The reconcile option seems to only be available here for 16 unless I'm missing it in 17 and 18. From 17 we have the Batch Payments tab on the bank reconciliation view.

Reply:
@dade-odoo Yes, you're right, I'll update it from 17.